docs/src/reference/types.md

```diff
@@ -630,6 +630,15 @@ Folds all items into a single value using an accumulator function.
   and one for an item.
 - returns: any
 
+### zip()
+Zips the array with another array. 
+If the two arrays are of unequal length, it will
+only zip up until the last element of the smaller array 
+and the rest will be ignored.
+
+- other: array (positional, required)
+  The other array which should be zipped with the current one.
```

Comment:
this is missing the return value. would also be good to mention that the array will consist of two-element arrays. (we could even make this variadic and have three/four/...-element arrays!, but that can also be a future addition)

could you also wrap the description at 80 columns?

Reply:
I added the return type. The wrapping should also be fine now.

And you are right, I had the same thought about extending it to n-elements, but it's a bit trickier since as far as I can see the Rust one only supports 2 by default. So if it's okay with you, I would leave it be as is for now and implement this in a separate PR.

Reply:
Oops sorry, I just noticed I reformatted the whole `types.md` file now. One second.